larray/core/array.py

```diff
@@ -5685,17 +5683,22 @@ def to_excel(self, filepath=None, sheet_name=None, position='A1',
                     if not os.path.isfile(filepath):
                         new_workbook = True
                     close = True
+            if overwrite_file:
```

Comment:
Solution is to replace 
if new_workbook:
    sheet = wb.sheets[0]
by
if overwrite_file:
    sheet = wb.sheets[0]